### ISSUE TITLE
RFP - JWPxp / JWPce - Dictionary MIJET (English to Japanese Language Dictionary)

### ISSUE THREAD

**Issue by animehunter123** [chocolatey-community/chocolatey-package-requests#736]
## Checklist

- [ ] The package I am requesting does not already exist on https://chocolatey.org/packages;
- [ ] There is no open issue for this package;
- [ ] The issue title starts 'RFP - ';
- [ ] The download URL is public and not locked behind a paywall / login;

## Package Details

Software project URL : https://mijet.eludevisibility.org/JWPxp/JWPxp.html
Direct download URL for the software / installer :  https://mijet.eludevisibility.org/JWPxp/JWPxp.7z
Software summary / short description: JWPxp / JWPce is a Japanese text editor with a Japanese/English Language Dictionary for students who study the language.

**Comment by animehunter123** [comment]
Hi everyone, I created this chocolatey package request. I think its very easy to get into the chocolatey package repositories. Can you guys add it? It will help a lot of us Japanese Language Learners.

**Comment by pauby** [comment]
@animehunter123 Just to be clear the Chocolatey Team don't add packages that are requested. This is a repository for current / new maintainers to create packages (or become maintainers of existing packages).

It's also a great excuse to learn how to create packages. Take a look at https://chocolatey.org/docs/create-packages for more information.

**Comment by animehunter123** [comment]
Thank you pauby. I will wait for the owner of the software to hopefully read this and allow it to be added into the choco repository. Thank you very much.

**Comment by pauby** [comment]
@animehunter123 I think I may not have been clear. Let me try and clear it up.

We don't need to wait for the software author to create a package. _Anybody_ can create a package for the Chocolatey Community Repository. What you've done is create a request for a new Chocolatey package to be created. The Chocolatey Team don't create packages themselves. Instead we ask people to create these requests and anybody can come along, see your request and create a package for it. That could be the author of the software, and it would be great if they would do that, but it could equally be you, me or anybody else.

If you are unable to create the package yourself then that is fine too. Somebody may come along and pick this up and create it.

Does that make sense?

**Comment by animehunter123** [comment]
Yes, that makes a lot of sense. I understand much better now, thank you for explaining this to me it makes so much more sense now.

**Comment by animehunter123** [comment]
Whew... it took me a few months!!! But I think I was able to make the package, tanos said we can use his zip url for now. Here is the git project I made:

https://github.com/animehunter123/jwpce

Do you know how we can add this to the chocolatey repositories?

**Comment by animehunter123** [comment]
I finished the nugget package, and it now fully installs and uninstalls. I think its ready to be uploaded to the chocolatey repository, what should we do next?

Here is our repository:
https://github.com/animehunter123/jwpce

**Comment by pauby** [comment]
@animehunter123 Have a look at the [docs](https://chocolatey.org/docs/create-packages).

**Comment by animehunter123** [comment]
Thanks Mr. Pauby, I will follow the docs to do your steps! 
![jwpce](https://user-images.githubusercontent.com/42163211/89867125-aa595480-dbeb-11ea-9744-a2d065a81321.png)

**Comment by animehunter123** [comment]
Hello Pauby, I pushed the package last week and it passed both of the automation tests. I follewed the docs, and everything looks ok. When will it be query'able via the webpage publically?

https://chocolatey.org/packages/jwpce  <--- here are the test results of the package.

**Comment by pauby** [comment]
@animehunter123 As the majority of moderators are volunteers life tends to get in the way of moderation unfortunately. Moderation can take anywhere from a few days to several weeks depending on the queue length and moderator availability. You can see the moderation queue, and the order of packages within it, by going to the packages page and change the filter from 'Normal View' to 'Moderator Queue'. One of the moderators will pick up your package as soon as possible.

**Comment by animehunter123** [comment]
Perfect, as always thank you very much for your support. I appreciate it very much. (This was actually the first thing I ever coded, so I was excited about it). I appreciate your help and the chocolatey team. Have a wonderful week.
